### DISCUSSION TITLE
Directories

### DISCUSSION THREAD

**Discussion by Reqteq** [mbround18/valheim-docker#249] [Q&A]
I just have a question regarding the volumes, I not a super linux user or anything but will using different directories from the ./valheim ones you specified in the compose file cause any problems? Or do you have some flexibility in where you mount the container volumes to?

Thanks

**Comment by mbround18** [comment]
You can mount it anywhere on the host side, it just has to be a valid path.
```
    volumes:
      - ./any/where/saves:/home/steam/.config/unity3d/IronGate/Valheim
      - ./any/where/server:/home/steam/valheim
      - ./any/where/backups:/home/steam/backups
```
